src/SwooleRequestHandlerRunner.php

```diff
@@ -39,6 +35,11 @@
  */
 class SwooleRequestHandlerRunner extends RequestHandlerRunner
 {
+    /**
+     * Default Process Name
+     */
+    private const DEFAULT_PROCESS_NAME = 'expressive';
```

Comment:
If you make this public, we can reference it in the `ConfigProvider` class; that's less magic in that class as a result.